src/ui/components/TestSuite/views/GroupedTestCases/Panel.tsx

```diff
@@ -72,7 +72,7 @@ export default function Panel() {
             className={styles.Attribute}
             icon="schedule"
             label={getTruncatedRelativeDate(recording.date)}
-            title={`${date.toLocaleDateString()} ${date.toLocaleTimeString()}`}
+            title={date.toLocaleString()}
```

Comment:
This is a more compact way of doing the same thing 🤷🏼

Reply:
Nice